package.json

```
@@ -4,14 +4,18 @@
   "private": true,
   "license": "MIT",
   "scripts": {
-    "assemble": "./assemble.js",
+    "assemble": "npm-run-all --serial assemble-local format",
+    "assemble-local": "npm-run-all --parallel -r worker assemble-cmd",
+    "assemble-cloud": "npm-run-all --parallel -r worker assemble-cloud-cmd",
```

Fixes cloud worker issue described bellow
```suggestion
    "assemble-cloud": "npm-run-all --parallel -r cloud-worker assemble-cloud-cmd",
```